src/views/trendingMedia.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from 'react';
+import { getTvShows } from "../utils/http.utils";
+import ToggleButtons from "../components/toggleButtons";
+import ResponsiveAppBar from '../components/navbar';
+import CircularProgress from '@mui/material/CircularProgress';
+
+export default function TrendingMedia() {
+    const [tvShows, setTvShows] = useState(null);
+    const [timeWindow, setTimeWindow] = useState('day');
+
+    useEffect(
+        () => { 
+            getTvShows(timeWindow)
+            .then((data) => {setTvShows(data.results)}) 
+        }, [timeWindow])
+
+    return (
+        <div>
+            <ResponsiveAppBar/>
+            <h1>Trending TV Shows</h1>
+            <ToggleButtons currentTimeWindow={timeWindow} setNewTimeWindow={setTimeWindow}/>
+            {tvShows && tvShows.length ? (
```

Comment:
This validation will show the `CircularProgress` if the number or `tvShows` is zero. Let's fix that so that we will show the `CircularProgress` when there no tvShow loaded yet and show a `'No TV Shows` message if the number of TV shows is zero.